test/parallel/test-runner-coverage.js

```diff
@@ -428,3 +428,72 @@ test('coverage with included and excluded files', skipIfNoInspector, () => {
   assert.strictEqual(result.status, 0);
   assert(!findCoverageFileForPid(result.pid));
 });
+
+test('coverage with a minimum line threshold', skipIfNoInspector, () => {
+  const fixture = fixtures.path('test-runner', 'coverage.js');
+  const args = [
+    '--experimental-test-coverage', '--test-coverage-min-line=80', '--test-reporter', 'tap',
+    fixture,
+  ];
+  const result = spawnSync(process.execPath, args);
+  const report = getTapCoverageFixtureReport();
+
+  if (common.isWindows) {
+    return report.replaceAll('/', '\\');
+  }
+
+  assert(result.stdout.toString().includes(report));
+  assert(result.stdout.toString().includes('Error: 78.35% line coverage does not meet threshold of 80%'));
+  assert.strictEqual(result.status, 1);
+  assert(!findCoverageFileForPid(result.pid));
+});
+
+test('coverage with a minimum branch threshold', skipIfNoInspector, () => {
+  const fixture = fixtures.path('test-runner', 'coverage.js');
+  const args = [
+    '--experimental-test-coverage', '--test-coverage-min-branch=80', '--test-reporter', 'tap',
+    fixture,
+  ];
+  const result = spawnSync(process.execPath, args);
+  const report = getTapCoverageFixtureReport();
+
+  if (common.isWindows) {
+    return report.replaceAll('/', '\\');
+  }
+
+  assert(result.stdout.toString().includes(report));
+  assert(result.stdout.toString().includes('Error: 43.75% branch coverage does not meet threshold of 80%'));
+  assert.strictEqual(result.status, 1);
+  assert(!findCoverageFileForPid(result.pid));
+});
+
+test('coverage with a minimum function threshold', skipIfNoInspector, () => {
+  const fixture = fixtures.path('test-runner', 'coverage.js');
+  const args = [
+    '--experimental-test-coverage', '--test-coverage-min-function=80', '--test-reporter', 'tap',
+    fixture,
+  ];
+  const result = spawnSync(process.execPath, args);
+  const report = getTapCoverageFixtureReport();
+
+  if (common.isWindows) {
+    return report.replaceAll('/', '\\');
+  }
+
+  assert(result.stdout.toString().includes(report));
+  assert(result.stdout.toString().includes('Error: 60.00% function coverage does not meet threshold of 80%'));
+  assert.strictEqual(result.status, 1);
+  assert(!findCoverageFileForPid(result.pid));
+});
+
+test('coverage with an out-of-range threshold', skipIfNoInspector, () => {
+  const fixture = fixtures.path('test-runner', 'coverage.js');
+  const args = [
+    '--experimental-test-coverage', '--test-coverage-min-branch=200', '--test-reporter', 'tap',
```

Comment:
```suggestion
    '--experimental-test-coverage', '--test-coverage-min-branch=101', '--test-reporter', 'tap',
```

Testing -1 would be good too.